common_functions.sh

```diff
@@ -393,6 +393,11 @@ function get_sums_for_build_arch() {
 			shasums_url=$(cat ${shasum_file} | grep "checksum_link" | head -1 | awk -F'"' '{ print $4 }');
 		fi
 		shasum=$(curl -Ls ${shasums_url} | sed -e 's/<[^>]*>//g' | awk '{ print $1 }');
+		# Sometimes shasum files are missing, check for error and ignore on error.
+		shasum_available=$(echo ${shasum} | grep -e "No" -e "Not");
+		if [ ! -z "${shasum_available}" ]; then
+			continue;
```

Comment:
Do we really want to ignore?  I'm not sure you want to pull in unverifiable binaries

Reply:
Sorry I meant "do not print" the arch if the shasum is not available. Have updated the comment to indicate that now.